### PR TITLE
Setup angular project for banana share link

### PR DESCRIPTION
**Pull request by JayCheck**
Initial skeleton project setup for share link.

### CODE REVIEW

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { TestBed } from '@angular/core/testing';
```
Comment (romandev) [inline src/app/app.component.spec.ts:6]: Please add this boilerplate in this file and all source codes

```
// Copyright 2022 The Triple Banana Authors. All rights reserved.
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    }).compileComponents();
+  });
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
+  it(`should have as title 'ShareLink'`, () => {
```
Comment (romandev) [inline src/app/app.component.spec.ts]: This looks unnecessary test

File: src/app/app.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+describe('AppComponent', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [
+        AppComponent
+      ],
+    }).compileComponents();
+  });
+
+  it('should create the app', () => {
```
Comment (romandev) [inline src/app/app.component.spec.ts]: Does this test make sense? We don't need to test a specific framework like Angular. I think we need to test our own logic. In this patch, we don't have any logic yet.

File: src/app/app.component.spec.ts (hunk 2 above)
Comment (JayCheck) [reply src/app/app.component.spec.ts]: it generated aytomatically by angular f/w will remove it

File: src/app/app.component.spec.ts (hunk 1 above)
Comment (JayCheck) [reply src/app/app.component.spec.ts:6]: will add it to app.component.spec.ts and app.component.ts

File: src/app/app.component.spec.ts (hunk 3 above)
Comment (JayCheck) [reply src/app/app.component.spec.ts]: it generated aytomatically by angular f/w will remove it

File: README.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+If user select "Share" menu of Triple Banana Browser and Copy or Share via apps,
+Shared link will connect to this share page.
+If the user have Triple Banana Browser in user's mobile phone, this link will be opened with Triple Banana Browser. If user does not have the browser, this link will open Google PlayStore to install Triple Banana Browser
+
+## Setup
+Project is using angular
+See https://angular.io/guide/setup-local
+
+## How to Run
+- $ng serve
```
Comment (romandev) [inline README.md]:
```
$ ng serve
```

File: src/app/app.module.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 The Triple Banana Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
```
Comment (romandev) [inline src/app/app.module.ts:5]: Please remove this line

File: src/app/app.component.html [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<!--
+    Copyright 2022 The Triple Banana Authors. All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at https://mozilla.org/MPL/2.0/.
```
Comment (romandev) [inline src/app/app.component.html]: For consistency, two blanks indentation

File: src/environments/environment.prod.ts [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export const environment = {
```
Comment (romandev) [inline src/environments/environment.prod.ts:6]: ditto

File: src/environments/environment.ts [hunk 8]
```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```
Comment (romandev) [inline src/environments/environment.ts]: ditto

File: src/styles.css [hunk 9]
```diff
@@ -0,0 +1 @@
+/* You can add global styles to this file, and also import other style files */
```
Comment (romandev) [inline src/styles.css]: Please add boilerplate code

File: src/polyfills.ts [hunk 10]
```diff
@@ -0,0 +1,53 @@
+/**
```
Comment (romandev) [inline src/polyfills.ts]: Please add boilerplate code

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**